api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java

```diff
@@ -36,9 +37,20 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
     }
 
     String authorizationHeader = request.getHeader(HttpHeaders.AUTHORIZATION);
+    if (authorizationHeader == null) {
+      Enumeration<String> headerNames = request.getHeaderNames();
+      String foundHeaders = "";
+      while (headerNames.hasMoreElements()) {
+        foundHeaders += " " + headerNames.nextElement();
+      }
+      log.warning("No " + HttpHeaders.AUTHORIZATION + " header found in request, found: "
```

Comment:
This is OK, but feels like something you added for debugging which we'll never need to use again?

(Once the client starts passing up authorization headers we'll never get here.)